crates/holochain/src/core/workflow/sys_validation_workflow/validation_query.rs

```diff
@@ -56,10 +56,15 @@ async fn get_ops_to_validate(
             ",
         );
     }
+    // TODO: There is a very unlikely chance that 10000 ops
+    // could all fail to validate and prevent validation from
+    // moving on but this is not easy to overcome.
+    // Once we impl abandoned this won't happen anyway.
```

Comment:
As @thedavidmeister mentioned, this is very likely to happen if someone decides to do it. You suggested a secondary ordering by validation attempts, is that easy to add?

Reply:
I added this :)